pytorch_examples/ex_blur_over_time.py

```diff
@@ -0,0 +1,30 @@
+# Example showing how to run FovVideoVDP on a numpy array with a video
+# The video is autogenerated in this example. See ex_aliasing.py for an example in which video is loaded from .mp4 files
+
+import os, sys
+import numpy as np
+import ex_utils as utils
+
+sys.path.append('..')
+from pyfvvdp import fvvdp
+import time
+
+
+# The frame to use for the video
+I_ref = utils.imread(os.path.join('..', 'matlab', 'examples', 'tree.jpg'))
```

Comment:
Please move all example images and video to <root>/example_content/ instead of having them hidden in Matlab and python example directories.